impl_core/src/main/java/io/opencensus/implcore/trace/SpanImpl.java

```diff
@@ -28,7 +28,7 @@
 import io.opencensus.trace.AttributeValue;
 import io.opencensus.trace.EndSpanOptions;
 import io.opencensus.trace.Link;
-import io.opencensus.trace.NetworkEvent;
+import io.opencensus.trace.MessageEvent;
```

Comment:
If the implementation uses MessageEvent, then it won't work with previous versions of the API.  I don't know if it's worth continuing to support that use case, though.  /cc @bogdandrutu

Reply:
Yes, that will be a problem. I'll reverse the implementation.